Casks/zoom-in.rb

```diff
@@ -0,0 +1,12 @@
+cask 'zoom-in' do
+  version '1.0.3'
+  sha256 'd27cd9bd969ef252537dd7d50175739021781e80219072159211786527e74864'
+
+  # github.com/nothingmagical/ZoomIn/ was verified as official when first introduced to the cask
+  url "https://github.com/nothingmagical/ZoomIn/releases/download/v#{version}/ZoomIn-#{version}.zip"
+  appcast 'https://github.com/nothingmagical/ZoomIn/releases.atom'
+  name 'Zoom In'
+  homepage 'https://zoom-in.app/'
+
+  app 'Zoom In.app'
+end
```

Comment:
To completely remove it from the system, you need to delete these:

```suggestion

  zap trash: [
               '~/Library/Application Scripts/com.nothingmagical.zoom-in',
               '~/Library/Containers/com.nothingmagical.zoom-in',
             ]
end
```